format_song_name.py

```
@@ -0,0 +1,70 @@
+    with open(filepath, 'r', encoding='utf-8') as f:
+        return json.load(f)
+
+def _save_json(filepath: str, data: Dict[str, Any]) -> None:
+    """安全保存 JSON 文件，保证中文不转码"""
+    with open(filepath, 'w', encoding='utf-8') as f:
+        # indent=4 增加可读性，可根据需要改为 indent=""
+        json.dump(data, f, ensure_ascii=False, indent=2)
+
+# ==================== 核心功能 ====================
+def format_missing_ids() -> None:
+    """补全缺失的曲目 ID，将 15~1343 之间缺失的 ID 填充为 None"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)